python/ql/src/experimental/semmle/python/frameworks/Django.qll

```diff
@@ -7,4 +7,260 @@ private import experimental.dataflow.DataFlow
 private import experimental.dataflow.RemoteFlowSources
 private import experimental.semmle.python.Concepts
 
-private module Django { }
+/**
+ * Provides models for the `django` PyPI package.
+ * See https://django.palletsprojects.com/en/1.1.x/.
```

Comment:
I would prefer to use the docstring I made in https://github.com/github/codeql/pull/4490/files#diff-7cbf166da1fec37a36d342fb500f780b1ef87b1d94a828f06e1d0dc616ddfaadR15 :stuck_out_tongue:  -- if you remove this change, we'll automatically get mine after a merge :+1:

Reply:
I might also take `django_attr` from there, and only have my one supported case. Then merging will be a bit smoother.